test/libsolidity/syntaxTests/functionTypes/valid_function_type_variables.sol

```diff
@@ -18,6 +18,6 @@ contract test {
     function(uint) pure internal public h = fh;
 }
 // ----
-// TypeError: (545-582): Internal or recursive type is not allowed for public state variables.
-// TypeError: (588-630): Internal or recursive type is not allowed for public state variables.
-// TypeError: (636-678): Internal or recursive type is not allowed for public state variables.
+// TypeError: (552-589): Internal or recursive type is not allowed for public state variables.
```

Comment:
This is not really related to this PR, but it seems weird to me that a test case called ``valid_function_type_variables`` expects several failures...

Reply:
Yes, I wonder when this was introduced.

Reply:
Ok, that would have been me on the 16th of March... :)

Can you please remove those variables? I saw there is a test that checks `internal public` to be invalid.

Reply:
Or perhaps just don't make them `public`.